cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/CxxCoverageSensor.java

```diff
@@ -247,7 +247,7 @@ private void saveZeroValueForResource(InputFile inputFile, SensorContext context
             }
           }
           measuresTotal.putAll(cacheCov.get(report.getAbsolutePath()));
-        } catch (EmptyReportException e) {
+        } catch (EmptyReportException | NullPointerException e) {
```

Comment:
should catch only EmptyReportException

Reply:
Unfortunately this sensor uses a mixture of exception and Boolean return value for handling the results :unamused:
By the way, we should not use null for empty results but an empty collection.

Reply:
cacheCov.get(report.getAbsolutePath()) returns null but should return an empty map ```Map<String, CoverageMeasures>>```